ccdproc/tests/test_ccddata.py

```diff
@@ -574,6 +574,7 @@ def test_header(ccd_data):
 def test_wcs(ccd_data):
     ccd_data.wcs = 5
     assert ccd_data.wcs == 5
+<<<<<<< HEAD
```

Comment:
I guess that's a leftover from your rebase @mwcraig mentioned. You need to delete this line.